packages/amplication-client/src/Components/EmptyState.scss

```diff
@@ -9,6 +9,7 @@ $svg-img-height: 200px;
   flex-direction: column;
   justify-content: center;
   align-items: center;
+  height: 100%;
```

Comment:
@sjcode99, thanks for your PR.
Changing the style of the component may impact the appearance in all places where the component is being used. 
Did you test it on other pages besides the commits page? 
Did you check how the alignment is achieved in other pages (without the addition of height:100%)? I see that the component had flex:1 - I assume this is the property that makes it happen in other pages